runtime/src/bank.rs

```diff
@@ -3514,6 +3545,7 @@ impl Bank {
                     let feature_set = self.feature_set.clone();
                     signature_count += u64::from(tx.message().header().num_required_signatures);
 
+                    // bprumo NOTE: Here's where the ComputeBudget is created
                     let mut compute_budget = self.compute_budget.unwrap_or_else(ComputeBudget::new);
```

Comment:
Maybe update the compute budget here with the number of bytes that can be allocated

Reply:
Yes, I can do that. It felt like the accounts data len wasn't really related to compute, but happy to move the max len into here.